seqr/views/apis/staff_api_tests.py

```diff
@@ -195,3 +202,75 @@ def test_elasticsearch_status(self, mock_elasticsearch, mock_get_mapping):
                                                       h="index,docs.count,store.size,creation.date.string")
         mock_es_client.cat.aliases.assert_called_with(format="json", h="alias,index")
         mock_get_mapping.assert_called_with(doc_type='variant,structural_variant')
+
+    def test_mme_details(self):
+        url = reverse(mme_details)
+        _check_login(self, url)
+
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(response_json.keys(), ['metrics', 'genesById', 'submissions'])
```

Comment:
please test some of the content of these. For example, test the metric keys are as expected, the length of the submissions are as expected, the keys in one of the submissions are as expected...

Reply:
I didn't test the content of the response because I didn't know where they come from. After I did some investigations, I think I can test the content of the metric dictionary, the length and keys of genesById, and the length of submissions. Do you think that is okay?